### PR TITLE
ocm - Update generator Prow config; Remove `grc-ui[-api]` (GRC)

### PR DESCRIPTION
**Pull request by dhaiducek**

### CODE REVIEW

**Comment by dhaiducek** [comment]
This PR is ready for review.

File: core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_pluginconfig.yaml [hunk 1]
```diff
@@ -2,8 +2,3 @@ plugins:
   open-cluster-management-io/policy-generator-plugin:
     plugins:
     - dco
-triggers:
```
Comment (gparvin) [inline core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_pluginconfig.yaml:5]: Were these changes intentional?

File: core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_prowconfig.yaml [hunk 2]
```diff
@@ -9,6 +9,7 @@ branch-protection:
               protect: true
               required_pull_request_reviews:
                 dismiss_stale_reviews: true
+                required_approving_review_count: 1
```
Comment (gparvin) [inline core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_prowconfig.yaml:12]: Did you mean to change this too?

File: core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_pluginconfig.yaml (hunk 1 above)
Comment (dhaiducek) [reply core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_pluginconfig.yaml:5]: Yes--`magic-mirror` doesn't open PRs upstream.

File: core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_prowconfig.yaml (hunk 2 above)
Comment (dhaiducek) [reply core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_prowconfig.yaml:12]: Yes--we want to gate upstream PRs with reviews.

**Review by gparvin — APPROVED** [review]
